src/types/references.types.ts

```diff
@@ -158,44 +56,43 @@ export enum ReferenceFieldNames {
 }
 
 export enum JournalArticleFieldNames {
-  TYPE = 'reference.journalArticle.type',
-  PUBLISHER = 'reference.journalArticle.publisher',
-  VOLUME = 'reference.journalArticle.volume',
-  ISSUE = 'reference.journalArticle.issue',
-  PAGES_FROM = 'reference.journalArticle.pagesFrom',
-  PAGES_TO = 'reference.journalArticle.pagesTo',
-  ARTICLE_NUMBER = 'reference.journalArticle.articleNumber',
-  PEER_REVIEW = 'reference.journalArticle.peerReview',
+  SUB_TYPE = 'entityDescription.publicationSubtype',
```

Comment:
kanskje litt teit å definere noen av disse feltene flere ganger (en gang for hver type), men det gjør det enklere å feks sette alle JournalArticle-felter som touched